src/auth/OAuthAuthenticator.js

```diff
@@ -206,57 +206,62 @@ OAuthAuthenticator.prototype.passwordGrant = function(userData, options, cb) {
 };
 
 /**
- * Sign in using a refresh token
+ * Exchange a refresh token
  *
  * @method    refreshToken
  * @memberOf  module:auth.OAuthAuthenticator.prototype
  *
  * @example <caption>
  *   Given a refresh token from a previous authentication request
- *   it will return a JSON with the access_token and id_token.
+ *   it will return a JSON with the access_token and id_token if
+ *   the openid scope was originally included.
  *   More information in the
  *   <a href="https://auth0.com/docs/api/authentication#refresh-token">
  *     API Docs
  *   </a>.
  * </caption>
  *
  * var data = {
- *   client_id: '{CLIENT_ID}', // Optional field.
  *   refresh_token: '{REFRESH_TOKEN}',
  * };
  *
- * auth0.oauth.refreshToken(data, function (err, userData) {
+ * auth0.oauth.refreshToken(data, function (err, data) {
  *   if (err) {
  *     // Handle error.
  *   }
  *
- *   console.log(userData);
+ *   console.log(data);
```

Comment:
Should we be leaving console logs in the codebase?

Reply:
@stevehobbsdev - Hard to tell here but that's a docblock code sample